dev/io.openliberty.mcp.internal/src/io/openliberty/mcp/internal/schemas/SchemaGenerator.java

```diff
@@ -177,7 +182,7 @@ public SchemaGenerationContext(SchemaCreationBlueprintRegistry blueprintRegistry
         public boolean registerSeen(Type type) {
             // If this is the first time we've seen this type, add it to the map with false,
             // If it's not the first time, set it to true
-            return typeMultiUse.compute(type, (k, v) -> v == null ? false : true);
+            return typeMultiUse.compute(type, (k, v) -> (v == null || (TypeUtility.hasGenericParams(type)) && type instanceof ParameterizedType) ? false : true);
```

Comment:
Unsure about this check - should we be detecting multi-use based on a type plus the resolution of any type variables?